### DISCUSSION TITLE
Are these t.Fatalf messages referencing the wrong error type?

### DISCUSSION THREAD

**Discussion by atc0005** [vmware/govmomi#2876] [Community Help] [answered]
https://github.com/vmware/govmomi/blob/62e5b573106a8dc1023bd57160ae1a91a02090a4/vim25/soap/client_test.go#L52-L55

https://github.com/vmware/govmomi/blob/62e5b573106a8dc1023bd57160ae1a91a02090a4/vim25/soap/client_test.go#L80-L83

Should the error mentioned instead be `errInvalidCACertificate`?

I assume I'm just misreading this, but figured it was worth asking just in case. I can file a PR if it needs to be corrected.

**Comment by embano1** [comment, marked as answer]
Yeah, this is likely a typo (missing CA)
